smoke-test/cdep.yml

```diff
@@ -36,6 +36,8 @@ dependencies:
 - compile: com.github.jomof:stb/tilemap_editor:0.0.0-rev6
 - compile: com.github.jomof:stb/truetype:0.0.0-rev6
 - compile: com.github.jomof:stb/voxel_render:0.0.0-rev6
-# - compile: com.github.gpx1000:v8:6.3.2163
-- compile: com.github.gpx1000:zlib:1.2.11
+#- compile: com.github.gpx1000:curl:7.56.0 # this has a problem specific to boringssl linking in the right order in ndk-build.  It otherwise works.
+#- compile: com.github.gpx1000:zlib:1.2.11 # this is a dependency of curl
```

Comment:
Possible to turn on zlib?

Reply:
Note that I turned zlib back on; but it will be redundant and might cause problems if it remains on when curl is also turned on as it's a direct dependency of curl; so would get added that way.
The potential for problem should already be addressed within CMake; but unsure of ndk-build.  Will address that potential with a different PR.